src/store/actions.js

```diff
@@ -309,4 +309,20 @@ export default {
 	updateGroupfolders(context, { groupfolder }) {
 		context.commit('UPDATE_GROUPFOLDERS', { groupfolder })
 	},
+	async addUsersFromCSV(context, { formData }) {
+		// console.debug('store.actions.addUsersFromCSV formData', formData)
+		const url = generateUrl('/apps/workspace/file/csv/import-data')
+		try {
+			const resp = await axios({
+				method: 'POST',
```

Comment:
Can you replace this called by `axios.post()`, please ?

Reply:
i tried this 
`const url = generateUrl('/apps/workspace/file/csv/import-data')`
`const resp = axios.post(url, { data: formData })`
but it doesn't work - the request body is empty object

Reply:
You don't need to add the `data` as key for the second argument :point_right: https://github.com/axios/axios#axiosposturl-data-config

You can write this code : 

```js
 const url = generateUrl('/apps/workspace/file/csv/import-data')
const resp = axios.post(url, { formData })
```

And it works :+1: 

For example with the `createSpace` function from spaceService.js : 

```js
/**
	* @param {string} spaceName it's a name for the space to create
	* @param {number} folderId it's the id of groupfolder
	* @param {object} vueInstance it's an instance of vue
	* @return {object}
	*/
export function createSpace(spaceName, folderId, vueInstance = undefined) {
	const result = axios.post(generateUrl('/apps/workspace/spaces'),
		{
			spaceName,
			folderId,
		})
		.then(resp => {
			if (typeof (resp.data) !== 'object') {
				throw new Error('Error when creating a space, it\'s not an object type.')
			}

			return resp.data
		})
		.catch(error => {
			if (typeof (vueInstance) !== 'undefined') {
				showNotificationError('Error to create a workspace', error.message, 5000)
			}
			throw new BadCreateError(error)
		})
	return result
}
```